javascript-forms/form-events.js

```diff
@@ -0,0 +1,22 @@
+function handleFocus(event) {
+  console.log('focus event was fired');
+  console.log('event.target.name:', event.target.name);
+}
+
+function handleBlur(event) {
+  console.log('blur event was fired');
+  console.log('event.target.name:', event.target.name);
+}
+
+function handleInput(event) {
+  console.log('event.target.name:', event.target.name);
+  console.log('event.target.value:', event.target.value);
+}
+
+var $name = document.getElementById('user-name');
+var $email = document.getElementById('user-email');
+var $message = document.getElementById('user-message');
+
+$name.addEventListener('focus', handleFocus);
+$email.addEventListener('blur', handleBlur);
+$message.addEventListener('input', handleInput);
```

Comment:
You should be applying all three of these event listeners to each form control element, not just one on each